Gopkg.toml

```diff
@@ -0,0 +1,38 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[prune]
+  go-tests = true
+  unused-packages = true
+
+[[constraint]]
+  name = "github.com/kyokomi/emoji"
+  version = "2.1.0"
+
+[[constraint]]
+  name = "github.com/pkg/errors"
```

Comment:
Where you use `errors` package ?

Reply:
@imatsiash-ib
I didn't, when i read about, how to use dependencies on windows, the example that was given is with the error package, so i forgot to delete it

Reply:
Please remove it.

Reply:
If you use dep - please push `vendor` folder to the github.